### PR TITLE
fix(client): remove use of jwt_access_token

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

**Notes:**
- API changes to be reverted before merge
  - Client to be deployed before API changes
  - Token has also been set to secure for production
- Should we add `res.setCookie('jwt_access_token', JRRTolkien, { httpOnly: true });` to `/get-session-user` on the api to force/update this for all?

### CODE REVIEW

File: api-server/src/server/utils/getSetAccessToken.js [hunk 1]
```diff
@@ -20,7 +20,9 @@ export function setAccessTokenToResponse(
 ) {
   const cookieConfig = {
     ...createCookieConfig(req),
-    maxAge: accessToken.ttl || 77760000000
+    maxAge: accessToken.ttl || 77760000000,
+    httpOnly: true,
+    secure: process.env.FREECODECAMP_NODE_ENV === 'production'
```
Comment (ojeytonwilliams) [inline api-server/src/server/utils/getSetAccessToken.js:25]:
```suggestion
    secure: process.env.FREECODECAMP_NODE_ENV !== 'development'
```
Not a big deal, but I think it's preferable to default to secure and make exceptions for specific cases.

**Review by ojeytonwilliams — APPROVED** [review]
The suggestion shouldn't be blocking, so LGTM :+1:

File: api-server/src/server/utils/getSetAccessToken.js (hunk 1 above)
Comment (ShaunSHamilton) [reply api-server/src/server/utils/getSetAccessToken.js:25]: You are supposed to be reviewing the client-side...

Comment (ojeytonwilliams) [reply api-server/src/server/utils/getSetAccessToken.js:25]: Ooops. I did that, too, and everything is fine there.

**Review by raisedadead — CHANGES_REQUESTED** [review]
Tests, please? 🙏🏽

**Pull request closed without merging**